src/main/frontend/src/components/aggregates/AggregateDialog.svelte

```diff
@@ -127,12 +110,12 @@
 	<ConsumerExchange bind:consumerExchangeName bind:receivers bind:methods />
 	<CardActions>
 		{#if editMode}
-			<Button class="mr-3" on:click={update} disabled={!valid}>Update</Button>
+			<Button class="mr-3" on:click={update} disabled={!valid}>Update</Button> <!--TODO: vlingo colors: class="primary-color", right now materialify overwrites disabled state-->
 		{:else}
-			<Button class="mr-3" on:click={add} disabled={!valid}>Add</Button>
+			<Button class="mr-3" on:click={add} disabled={!valid}>Add</Button> <!--TODO: vlingo colors-->
 		{/if}
-		<Button on:click={() => dialogActive = !dialogActive}>Cancel</Button>
+		<Button outlined on:click={closeDialog}>Cancel</Button>
 		<span style="width: 100%;"></span>
-		<Button on:click={reset}>Reset</Button>
+		<Button outlined class="red red-text" on:click={clearFields}>Clear Fields</Button>
```

Comment:
Colors now warn a bit better, also "Reset" sounded like in editMode you get back to before-the-edit, but we just clear the fields, so I renamed it